src/test/java/uk/gov/pay/directdebit/webhook/gocardless/services/GoCardlessPaymentHandlerTest.java

```diff
@@ -216,19 +226,4 @@ public void handle_onCreatePaymentGoCardlessEvent_shouldThrowExceptionWhenEventH
         goCardlessPaymentHandler.handle(goCardlessEvent);
     }
 
-    @Test
-    public void handle_onCreatePaymentGoCardlessEvent_shouldThrowPaymentNotFoundException() {
```

Comment:
Why is this no longer needed?

Reply:
The exception is now thrown by the `PaymentQueryService` and there’s a unit test for that there.